setup.py

```diff
@@ -356,6 +363,9 @@
     'obspy.plugin.inventory.CSS': [
         'writeFormat = obspy.io.css.station:_write_css',
         ],
+    'obspy.plugin.inventory.SEISAN': [
+        'writeFormat = obspy.io.nordic.core:inventory_to_seisan',
```

Comment:
This PR is solely about Nordic format, however, this was a little addition I had in EQcorrscan - it was suggested I could include it and plug it in as above (although naming isn't clear, it isn't a clear format in my mind, and this function doesn't make the whole file, there are other elements to it).  This function was just convenience for me.  Happy to remove and keep to myself?